$NetBSD: patch-configure,v 1.3 2024/08/09 12:02:45 adam Exp $

Check if putc_unlocked(); Fixes "duplicate symbol '___sputc'" on Darwin.
https://dev.gnupg.org/T7246

--- configure.orig	2024-08-09 11:49:20.106366280 +0000
+++ configure
@@ -15241,6 +15241,12 @@ then :
   printf "%s\n" "#define HAVE_GETRLIMIT 1" >>confdefs.h
 
 fi
+ac_fn_c_check_func "$LINENO" "putc_unlocked" "ac_cv_func_getrlimit"
+if test "x$ac_cv_func_getrlimit" = xyes
+then :
+  printf "%s\n" "#define HAVE_PUTC_UNLOCKED 1" >>confdefs.h
+
+fi
 
 
 # If we didn't find inet_pton, it might be in -lsocket (which might
